vck-rqes/src/commonMain/kotlin/at/asitplus/wallet/lib/RqesWalletService.kt

```diff
@@ -0,0 +1,92 @@
+package at.asitplus.wallet.lib
+
+import at.asitplus.openid.AuthenticationRequestParameters
+import at.asitplus.openid.AuthorizationDetails
+import at.asitplus.openid.CscAuthenticationRequestParameters
+import at.asitplus.openid.IssuerMetadata
+import at.asitplus.openid.OAuth2AuthorizationServerMetadata
+import at.asitplus.openid.OpenIdConstants.CODE_CHALLENGE_METHOD_SHA256
+import at.asitplus.openid.OpenIdConstants.GRANT_TYPE_CODE
+import at.asitplus.rqes.SignatureRequestParameters
+import at.asitplus.rqes.CSCSignatureRequestParameters
+import at.asitplus.rqes.RqesConstants
+import at.asitplus.rqes.SignDocParameters
+import at.asitplus.rqes.SignHashParameters
+import at.asitplus.rqes.enums.SignatureFormat
+import at.asitplus.signum.indispensable.X509SignatureAlgorithm
+import at.asitplus.wallet.lib.oauth2.OAuth2Client
+import com.benasher44.uuid.uuid4
+
+class RqesWalletService(
+    private val clientId: String = "https://wallet.a-sit.at/app",
+    private val redirectUrl: String = "$clientId/callback",
+    private val oauth2Client: OAuth2Client = OAuth2Client(clientId = clientId, redirectUrl = redirectUrl),
+) {
+
+    //TODO see below
+//    suspend fun createOAuth2AuthenticationRequest(
+//        rqesRequest: SignatureRequestParameters,
+//        credentialId: ByteArray,
+//    ): AuthenticationRequestParameters =
+//        oauth2Client.createAuthRequest(
+//            state = uuid4().toString(),
+//            authorizationDetails = setOf(rqesRequest.toAuthorizationDetails()),
+//            scope = RqesConstants.SCOPE,
+//            credentialId = credentialId,
+//        )
```

Comment:
Move it to an issue in some way or resolve it

Reply:
The `RqesWalletService` will be finished in #127